### PR TITLE
add resource type "kubeletconfig"

### PR DESCRIPTION
**Pull request by lyman9966**
add cluster resource type "kubeletconfig", and test log: https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3-smoke/2371/console [ `And I ensure "kubelet-set-negative" kubeletconfig is deleted after scenario` test pass ]

@jhou1 @sunilcio @weinliu  Can you help to review? Thanks

### CODE REVIEW

File: lib/openshift/kubelet_config.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require 'openshift/cluster_resource'
+
+module BushSlicer
+  class KubeletConfig < ClusterResource
+    RESOURCE = "kubeletconfigs"
```
Comment (akostadinov) [inline lib/openshift/kubelet_config.rb:5]: full api name is missing. Could you paste `oc get -o yaml` and I will help you get it right?

Comment (lyman9966) [reply lib/openshift/kubelet_config.rb:5]: @akostadinov thanks, please refer to :
# oc get kubeletconfig custom-kubelet-test -o yaml 
apiVersion: machineconfiguration.openshift.io/v1
kind: KubeletConfig
metadata:
  creationTimestamp: "2020-09-27T03:33:35Z"
  finalizers:
  - a7ff707c-ccc2-4268-9fe4-4a764c549a36
  generation: 1
  managedFields:
  - apiVersion: machineconfiguration.openshift.io/v1
    fieldsType: FieldsV1
    fieldsV1:
      f:spec:
        .: {}
        f:kubeletConfig:
          .: {}
          f:maxPods: {}
        f:machineConfigPoolSelector:
          .: {}
          f:matchLabels:
            .: {}
            f:custom-kubelet: {}
    manager: oc
    operation: Update
    time: "2020-09-27T03:33:35Z"
  - apiVersion: machineconfiguration.openshift.io/v1
    fieldsType: FieldsV1
    fieldsV1:
      f:metadata:
        f:finalizers:
          .: {}
          v:"a7ff707c-ccc2-4268-9fe4-4a764c549a36": {}
      f:status:
        .: {}
        f:conditions: {}
    manager: machine-config-controller
    operation: Update
    time: "2020-09-27T03:33:36Z"
  name: custom-kubelet-test
  resourceVersion: "2762882"
  selfLink: /apis/machineconfiguration.openshift.io/v1/kubeletconfigs/custom-kubelet-test
  uid: 48f2e490-719a-4579-a80f-90d0cd2717d6
spec:
  kubeletConfig:
    maxPods: 249
  machineConfigPoolSelector:
    matchLabels:
      custom-kubelet: test-pods
status:
  conditions:
  - lastTransitionTime: "2020-09-27T03:33:36Z"
    message: Success
    status: "True"
    type: Success

Comment (akostadinov) [reply lib/openshift/kubelet_config.rb:5]: It should be `kubeletconfigs.machineconfiguration.openshift.io`

**Pull request merged**